Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG DOCKER_BASE_IMAGE
+FROM $DOCKER_BASE_IMAGE
+
+ARG VCS_REF
+ARG BUILD_DATE
+LABEL \
+    maintainer="https://ocr-d.de/kontakt" \
+    org.label-schema.vcs-ref=$VCS_REF \
+    org.label-schema.vcs-url="https://github.com/qurator-spk/eynollah" \
+    org.label-schema.build-date=$BUILD_DATE
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV PYTHONIOENCODING=utf8
+ENV XDG_DATA_HOME=/usr/local/share
+
+WORKDIR /build-eynollah
+COPY qurator/ ./qurator
```

Comment:
> COPY qurator/ ./qurator

Is this still needed/correct after changing the package layout?

Reply:
Argh, I wrote this against the outdated version in Gitlab. Thanks for spotting. I'll update.